_posts/2016-02-18-starter-issues.md

```diff
@@ -0,0 +1,112 @@
+```
+
+Then we ask the user to commit and push the tests. This way the commit will trigger our continuous integration tests on [Travis CI](https://travis-ci.org) and will add a red **×** to the commit, so we can be sure that tests are failing as expected. Here’s [an example](https://github.com/hoodiehq/pouchdb-hoodie-api/pull/100/commits) of what I mean.
+
+```
+- [ ] Make sure that tests fail when run with `npm test`. Commit with `test: myLib.myMethod` and push
+```
+
+Now comes the fun, the “actual” work, and all it takes is an item describing it. If helpful, we link to the right place and explain how the feature / bug fix can be implemented. But it’s okay to keep it short and leave something for the contributor to figure out. It’s fun after all, and at this point the contributor should feel comfortable to ask for help if they get stuck. So often time we simply do:
+
+```
+- [ ] Implement :)
+- [ ] Make sure tests pass when run with `npm test`. Commit with `feat: myLib.myMethod` and push
+```
+
+The work is done. Now someone else needs to review and merge the work.
+
+```
+- [ ] Someone else: review & merge
+```
+
+During the review it often turns out that something is wrong or missing, so the pull request needs amendments. The contributor can simply add more commits as long as needed. Once they are done, they can either squash the commits themselves if they know how to do it, or the reviewer does it for them. Very important when the reviewer is cleaning up the commit messages is to keep the original author. Nothing is more disappointing than a first pull request accidentally “stolen” by a reviewer. A custom author can be set with `git commit --author="Pat Doe <pat@example.com>""`
+
+At the end of the issue we add ways to contact us, usually something like this:
+
+```
+Ping us in the [Hoodie Chat](http://hood.ie/chat/) or on [Twitter](https://twitter.com/hoodiehq/) if you have any questions :)
+```
+
+Please let us know if you have any suggestions on how we can further improve our starter issues – we love hearing from you <3
+
+### Beyond the extra mile
+
+If the issue involves writing a test, then you can create it yourself. Comment it out and push it to master. Then instead of asking the new contributor to figure out how to write the test, all the need to do is to uncomment it and see the tests fail. This idea comes from Kent C. Dodds who wrote about it in his piece on
+[First Timers Only](https://medium.com/@kentcdodds/first-timers-only-78281ea47455#.nkxl3r9p6) – one of our favourites :)
+
+### Benefits
+
+The benefits of starter issues go both ways. For people new to the project, it lowers the barrier and makes contributing more fun. For maintainers, it’s a great way to empower more people to do code reviews, as it’s just checking off the list. More importantly, it is our hope that creating starter issues will become a part of the Open Source culture and a great format for community engagement.
```

Comment:
"check items off of" Is there a way to rephrase this without the "off of" at the end? It’s a bit hard to read to me. Maybe we can just leave out the last "of"?